lib/hooks/useToast.ts

```diff
@@ -0,0 +1,189 @@
+// Inspired by react-hot-toast library
+import * as React from "react";
+
+import { type ToastProps, ToastActionElement } from "components/atoms/Toast/toast";
+
+const TOAST_LIMIT = 1;
+const TOAST_REMOVE_DELAY = 1000;
+
+type ToasterToast = ToastProps & {
+  id: string
+  title?: React.ReactNode
+  description?: React.ReactNode
```

Comment:
`description` being optional; Is it allowed design-wise to have a toast with only `title` and no `description`? That sounds like it can cause inconsistency between when 1 developer would print 1 line as description and another dev would print 1 line as title, and title & description don't look the same.

Reply:
Agree with this

Reply:
Nice catch!... I should make the description require and leave the title optional ✅